lord/time_lord_test.go

```diff
@@ -110,89 +141,103 @@ var _ = DescribeTable("A range without a previous time", (testCase).Run,
 		stop:   "4:00 AM +0000",
 		now:    "3:00 AM +0000",
 		result: true,
+		latest: expectedTime{hour: 2},
 	}),
 	Entry("between the start and stop time down to the minute", testCase{
 		start:  "2:01 AM +0000",
 		stop:   "2:03 AM +0000",
 		now:    "2:02 AM +0000",
 		result: true,
+		latest: expectedTime{hour: 2, minute: 1},
 	}),
 	Entry("not between the start and stop time", testCase{
 		start:  "2:00 AM +0000",
 		stop:   "4:00 AM +0000",
 		now:    "5:00 AM +0000",
 		result: false,
+		latest: expectedTime{isZero: true},
 	}),
 	Entry("after the stop time, down to the minute", testCase{
 		start:  "2:00 AM +0000",
 		stop:   "4:00 AM +0000",
 		now:    "4:10 AM +0000",
 		result: false,
+		latest: expectedTime{isZero: true},
 	}),
 	Entry("before the start time, down to the minute", testCase{
 		start:  "11:07 AM +0000",
 		stop:   "11:10 AM +0000",
 		now:    "11:05 AM +0000",
 		result: false,
+		latest: expectedTime{isZero: true},
 	}),
 	Entry("one nanosecond before the start time", testCase{
 		start:     "3:04 AM +0000",
 		stop:      "3:07 AM +0000",
 		now:       "3:03 AM +0000",
 		extraTime: time.Minute - time.Nanosecond,
 		result:    false,
+		latest:    expectedTime{isZero: true},
 	}),
 	Entry("equal to the start time", testCase{
 		start:  "3:04 AM +0000",
 		stop:   "3:07 AM +0000",
 		now:    "3:04 AM +0000",
 		result: true,
+		latest: expectedTime{hour: 3, minute: 4},
 	}),
 	Entry("one nanosecond before the stop time", testCase{
 		start:     "3:04 AM +0000",
 		stop:      "3:07 AM +0000",
 		now:       "3:06 AM +0000",
 		extraTime: time.Minute - time.Nanosecond,
 		result:    true,
+		latest:    expectedTime{hour: 3, minute: 4},
 	}),
 	Entry("equal to the stop time", testCase{
 		start:  "3:04 AM +0000",
 		stop:   "3:07 AM +0000",
 		now:    "3:07 AM +0000",
 		result: false,
+		latest: expectedTime{isZero: true},
 	}),
 
 	Entry("between the start and stop time but the stop time is before the start time, spanning more than a day", testCase{
 		start:  "5:00 AM +0000",
 		stop:   "1:00 AM +0000",
 		now:    "6:00 AM +0000",
 		result: true,
+		latest: expectedTime{hour: 5},
 	}),
 	Entry("between the start and stop time but the stop time is before the start time, spanning half a day", testCase{
 		start:  "8:00 PM +0000",
 		stop:   "8:00 AM +0000",
 		now:    "1:00 AM +0000",
 		result: true,
+		latest: expectedTime{hour: 20, weekday: time.Saturday},
```

Comment:
Note to self: this, along with the other examples, expects Saturday because `nowDay` defaults to 0, i.e. Sunday, and we're returning 8PM on the previous day, i.e. Saturday.